modules/gdnative/godot/godot_image.cpp

```diff
@@ -105,6 +105,43 @@ int GDAPI godot_image_get_mipmap_count(const godot_image *p_img) {
 	return img->get_mipmap_count();
 }
 
+void GDAPI godot_image_blit_rect(godot_image *p_img, godot_image *src) {
+
+}
+
+void GDAPI godot_image_compressed(godot_image *p_img, int p_mode) {
+	Image *img = (Image *)p_img;
+	memnew_placement_custom(img, Image, img->compressed(p_mode));
```

Comment:
`memnew_placement_custom` is only used to *construct* the objects. If you just want to use them you can do

```cpp
Image *img = (Image *)p_img;
img->compressed(p_mode);
```